src/Components/Filter.js

```diff
@@ -0,0 +1,53 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+
+function CategoryFilter(props) {
+  const [value1, setvalue] = useState({ value: 'All' });
+  const { clickHandler } = props;
+  const categories = ['Action', 'Biography', 'History', 'Horror', 'Kids', 'Learning', 'Sci-Fi'];
+
+  const changevalue = (event) => {
+    setvalue({ value: event });
+  };
+
+  const middlefunc = (e) => {
+    clickHandler(e.target.value);
+    changevalue(e.target.value);
+  };
```

Comment:
- [ ] CategoryFilter is a presentational component [screenshot from Pathwright](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/bookstore/milestone4/presentationalCategoryFilter.png). This means that it cannot be used to change the state. It should only render the select box with filter categories. Kindly let us update this.